app/src/App.svelte

```diff
@@ -40,26 +45,37 @@
     "/search": Search,
     "/feed": Feed,
     "/projects": Projects,
-    "/projects/:domain/:name/*": Project,
+    "/projects/new": CreateProject,
+    "/projects/:id/*": Project,
     "/design-system": DesignSystem,
     "/wallet": Wallet,
     "/profile": Profile,
+    "/help": Help,
     "*": NotFound
   };
-</script>
 
-<style>
-  .container {
-    position: relative;
-    left: var(--slim-sidebar-width);
-    width: calc(100vw - var(--slim-sidebar-width));
-    overflow-x: hidden;
-  }
-</style>
+  hotkeys("d,shift+d", () => {
```

Comment:
Maybe let's stick to one modifier for now and not use single characters to avoid surprises.
```suggestion
  hotkeys("shift+d", () => {
```